src/styles/partials/components/_accordion-dark.scss

```diff
@@ -25,6 +25,10 @@
       a {
         color: $white;
       }
+      h2 {
```

Comment:
these styles aren't dark specific, so they should go into the _accordion.scss file. If for some reason this is an issue, please let me know and we can talk through this.